__tests__/components/App.test.js

```diff
@@ -88,50 +88,68 @@ describe('#routes', () => {
       expect(component.find(HomePage).length).toEqual(1)
     })
 
-    it('renders the Editor component when "/editor" is visited with resource', () => {
-      const component = renderRoutes('/editor')
-
-      expect(component.find(Editor).length).toEqual(1)
-    })
-
-    it('renders the Search component when "/search" is visited', () => {
+    it('renders the HomePage component when "/search" is visited', () => {
       const component = renderRoutes('/search')
 
-      expect(component.find(Search).length).toEqual(1)
+      expect(component.find(HomePage).length).toEqual(1)
     })
 
-    it('renders the Menu component when "/menu" is visited', () => {
-      const component = renderRoutes('/menu')
+    it('renders the HomePage for unknown path', () => {
+      const component = renderRoutes('/blah')
 
-      expect(component.find(CanvasMenu).length).toEqual(1)
+      expect(component.find(HomePage).length).toEqual(1)
     })
 
-    it('does not render ImportResourceTemplate component (even if user visits "/templates"), since user is not logged in', () => {
-      const component = renderRoutes('/templates')
-
-      expect(component.find(ImportResourceTemplate).length).toEqual(0)
-    })
 
-    it('renders a 404 for an unknown path', () => {
-      const component = renderRoutes('/blah')
-
-      expect(component.contains(<h1>404</h1>)).toEqual(true)
-    })
+    // it('renders the Editor component when "/editor" is visited with resource', () => {
+    //   const component = renderRoutes('/editor')
+    //
+    //   expect(component.find(Editor).length).toEqual(1)
+    // })
+    //
+    // it('renders the Search component when "/search" is visited', () => {
+    //   const component = renderRoutes('/search')
+    //
+    //   expect(component.find(Search).length).toEqual(1)
+    // })
+    //
+    // it('renders the Menu component when "/menu" is visited', () => {
+    //   const component = renderRoutes('/menu')
+    //
+    //   expect(component.find(CanvasMenu).length).toEqual(1)
+    // })
+    //
+    // it('does not render ImportResourceTemplate component (even if user visits "/templates"), since user is not logged in', () => {
+    //   const component = renderRoutes('/templates')
+    //
+    //   expect(component.find(ImportResourceTemplate).length).toEqual(0)
+    // })
+    //
+    // it('renders a 404 for an unknown path', () => {
+    //   const component = renderRoutes('/blah')
+    //
+    //   expect(component.contains(<h1>404</h1>)).toEqual(true)
+    // })
```

Comment:
if this PR goes in, these commented out lines should go away? or??

Reply:
Uggh, yes. Removed.